trajopt/CMakeLists.txt

```diff
@@ -46,6 +46,20 @@ target_include_directories(${PROJECT_NAME} SYSTEM PUBLIC
 
 trajopt_configure_package(${PROJECT_NAME})
 
+# Install test data directory for use in other packages
```

Comment:
If test data will always be installed you could add the compiler definition to the main target.

Reply:
What is your preference? You are saying I should add `target_compile_definitions(${PROJECT_NAME} SHARED TRAJOPT_DIR="${CMAKE_INSTALL_PREFIX}/share/${PROJECT_NAME}")` to the target above?

Reply:
https://cmake.org/cmake/help/latest/variable/CMAKE_CURRENT_SOURCE_DIR.html